code/datums/components/customizable_reagent_holder.dm

```diff
@@ -123,6 +123,11 @@
 /datum/component/customizable_reagent_holder/proc/customizable_attack(datum/source, obj/ingredient, mob/attacker, silent = FALSE, force = FALSE)
 	SIGNAL_HANDLER
 
+	// This is for stuff like adding flour from a flour sack into a bowl, we handle the transfer of the reagent elsewhere, but we shouldn't regard it beyond some user feedback.
+	if (istype(ingredient, /obj/item/reagent_containers))
+		attacker.balloon_alert(attacker, "transferring...")
+		return
+
 	if (!valid_ingredient(ingredient))
 		attacker.balloon_alert(attacker, "doesn't go on that!")
 		return
```

Comment:
```suggestion
	if (!valid_ingredient(ingredient))
		if (istype(ingredient, /obj/item/reagent_containers))
			attacker.balloon_alert(attacker, "transferring...")
			return
		attacker.balloon_alert(attacker, "doesn't go on that!")
		return
```
Try something like above, that way you don't make some poor coder cry in 2 years when they make an edible beaker that somehow doesn't obey the TRAIT_ODD_CUSTOMIZABLE_FOOD_INGREDIENT

Also why not check for DRAINABLE on the reagent datum? That catches all pouring attempts, instead of just reagent_container subtypes (and it also doesn't make sense to give pills, patches, etc the "transferring..." warning